src/coreclr/vm/qcallentrypoints.cpp

```diff
@@ -320,6 +322,16 @@ static const Entry s_QCall[] =
     DllImportEntry(ComWeakRefToObject)
     DllImportEntry(ObjectToComWeakRef)
 #endif
+#ifdef FEATURE_EH_FUNCLETS
+    DllImportEntry(RhpSfiInit)
```

Comment:
We should avoid introducing Rh... naming convention in CoreCLR. We ported some Rh... helpers in the past, but it did not look good and it did not age well. They have been renamed since then.

I hope that we will get rid of Rh names in NAOT at some point.